release/models/platform/openconfig-platform-linecard.yang

```diff
@@ -23,7 +23,14 @@ module openconfig-platform-linecard {
     "This module defines data related to LINECARD components in
     the openconfig-platform model";
 
-  oc-ext:openconfig-version "0.1.2";
+  oc-ext:openconfig-version "0.1.3";
```

Comment:
due to deletion of leaf (breaking change), this is now a new, major release, so version should become "1.0.0"